tools/cli/commands.js

```diff
@@ -1203,71 +1203,85 @@ main.registerCommand({
     toIgnore.push(/(\.html|\.js|\.css)/);
   }
 
-  try {
-    // Prototype option should use local skeleton.
-    // Maybe we should use a different skeleton for prototype
-    if (options.prototype) throw new Error("Using prototype option");
-    // if using the release option we should use the default skeleton
-    // using it as it was before 2.x
-    if (release.explicit) throw new Error("Using release option");
-
-    await setupExampleByURL(`https://github.com/meteor/skel-${skeleton}`);
-  } catch (e) {
+  const copyFromLocalSkeleton = async () => {
+    await files.cp_r(
+      skeletonPath,
+      appPath,
+      {
+        transformFilename: function (f) {
+          return transform(f);
+        },
+        transformContents: function (contents, f) {
+          // check if this app is just for prototyping if it is then we need to add autopublish and insecure in the packages file
+          if (/packages/.test(f)) {
```

Comment:
i think it isnt compatible with windows because of `/` 
It would be nice to test it first

Reply:
This is not new code, just a refactor.

Why it won't work in Windows? This is not a path `/`, is just a regex delimiter, and Node/JS syntax is cross-os compatible.